exchanges/bitfinex/bitfinex_websocket.go

```diff
@@ -33,6 +33,10 @@ import (
 	"github.com/thrasher-corp/gocryptotrader/log"
 )
 
+var (
+	errParsingWSField = errors.New("error parsing WS field")
+)
```

Comment:
Not so convinced, oddly, since it feels like this is a block of error vars, that happens to only contain one.
I want devs to add to it, essentially.

Did linter or editor highlight this?

Reply:
I am just reducing lines. Something like error blocks is usually the responsibility of the reviewers to maintain/enforce. But will leave it up to you if you want to commit that the suggestion.